decisions/wc_religion_decisions.txt

```diff
@@ -217,6 +217,7 @@ decisions = {
 			is_incapable = no
 
 			religion_group = primitive_group
+			NOT = { religion = cult_of_the_candle }
```

Comment:
I don't think it makes sense for kobolds to have these anyway. I don't really think anyone besides feteshistic religion should have it, but it may act as replacer for now, I guess?

Reply:
So it's basically replacement? I think it's a bad idea, I like Primitive Rites more, it has more options and possibilities.